resolwe/flow/executors/docker/run.py

```diff
@@ -216,22 +216,35 @@ def new_volume(kind, base_dir_name, volume, path=None, read_only=True):
         # A non-login Bash shell should be used here (a subshell will be spawned later).
         command_args['shell'] = '/bin/bash'
 
-        pull_command = '{command} pull {container_image}'.format(**command_args)
+        # Check if image exists locally. If not, command will exit with non-zero returncode
+        check_command = '{command} image inspect {container_image}'.format(**command_args)
 
-        logger.info("Pulling docker image: {}".format(command_args['container_image']))
+        logger.info("Checking existence of docker image: {}".format(command_args['container_image']))
```

Comment:
This should be a debug log as the following command is local and should execute instantly.